pip/pep425tags.py

```diff
@@ -189,6 +236,11 @@ def get_supported(versions=None, noarch=False):
             else:
                 # arch pattern didn't match (?!)
                 arches = [arch]
+        elif sys.platform == 'linux':
+            if is_manylinux1_compatible():
+                arches = [arch, arch.replace('linux', 'manylinux1')]
```

Comment:
I don't know, either what the effect is or what the preferred behavior is... To check what the effect is I guess it would be easy to check (you can trivially run pip out of a source checkout by running `python -m pip`while in the root of the checkout). And in terms of what behavior we want, I think it's pretty arbitrary really -- the case where it comes up is very unlikely, and in this unlikely case, I can't think of any strong reason to prefer one outcome or another.